src/hocs/ec-with-sorting/ec-with-sorting.js

```diff
@@ -64,8 +65,9 @@ const withSorting = createHOCc({
     },
   },
   listeners: {
-    sort(columnName) {
-      this.sortBy(columnName);
+    sort(column) {
+      const { name: columnName, sortCycle } = column;
+      this.sortBy(columnName, sortCycle);
```

Comment:
I'm thinking about creating and exporting an enum sort-direction-cycle.js:
```
export const LOWEST_FIRST = [ASC, DESC]
export const HIGHEST_FIRST= [DESC, ASC]
```

so then we can do in EBO:
```
columns: [{
  name: 'currency',
  title: 'Currency column',
  sortable: true,
  sortCycle: SortDirectionCycle.HIGHEST_FIRST,
}]
```
Or we can declare then in EBO only but more specific:
```
export const NUMBERS = [DESC, ASC]
```
Or we can use both approaches:
```
export const NUMBERS = HIGHEST_FIRST // HIGHEST_FIRST comes from chameleon, NUMBERS is declared in EBO.
```

You can also reuse the enum for default of value of sortCycle prop:
```
props: {
  sortCycle: {
    default = () => SortDirectionCycle.LOWEST_FIRST
}
}

Reply:
I really like the idea thanks

- [ ] Declare enum